provider_test.go

```diff
@@ -420,11 +421,21 @@ func TestSetRecords(t *testing.T) {
 		t.Fatalf("p.SetRecords() unexpected diff [-want +got]: %s", diff)
 	}
 
+	unchanged := []libdns.Record{
+		{
+			Type:  "TXT",
+			Name:  "sub",
+			Value: `will stay the same`,
+			TTL:   time.Second * 3600,
+		},
+	}
+	wantCurrent := append(records, unchanged...)
+
```

Comment:
In the other cases where a change is verified, `want` is used as the variable name and the list of records is spelled out fully. This is a bit annoying here, because there are very many records it might even be cleaner the way you propose here. I am torn, but I tend towards consistency.

Reply:
Changed the code to explicitly list all expected records.